doc/cephadm/upgrade.rst

```diff
@@ -56,7 +56,7 @@ Before you use cephadm to upgrade Ceph, verify that all hosts are currently onli
 
    ceph -s
 
-To upgrade (or downgrade) to a specific release, run the following command:
+To upgrade to a specific release, run the following command:
 
 .. prompt:: bash #
```

Comment:
Good idea.  I think Zac favors "run a command of the following form"

Reply:
changed to this format for both this line and the one below